src/jit/compiler.hpp

```diff
@@ -632,7 +632,7 @@ bool                isRegParamType(var_types type)
 #endif // !_TARGET_X86_
 }
 
-#ifdef _TARGET_AMD64_
+#if defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_)
```

Comment:
This method doesn't seem to be handling the case of 16-byte struct passed in two argument registers.  For an example see x64 unix logic in this routine.